virtual_ecosystem/models/hydrology/hydrology_tools.py

```diff
@@ -107,15 +107,16 @@ def setup_hydrology_input_current_timestep(
         seed=seed,
     )
 
-    for out_var, in_var in (
-        ("subcanopy_temperature", "air_temperature"),
-        ("subcanopy_humidity", "relative_humidity"),
-        ("subcanopy_wind_speed", "wind_speed"),
-        ("subcanopy_pressure", "atmospheric_pressure"),
-    ):
-        output[out_var] = (
-            data[in_var].isel(layers=layer_roles.index("subcanopy")).to_numpy()
-        )
+    # VIVI: these to go?
```

Comment:
We still need these values, but I will rename them to 'surface' for now until we have sorted the interpolation to 1.5m